app/kumactl/cmd/install/install_dns.go

```diff
@@ -52,30 +57,34 @@ This command requires that the KUBECONFIG environment is set`,
 
```

Comment:
please add error handling to `NewForConfig`. Yesterday we stumbled upon error there and it was ignored.